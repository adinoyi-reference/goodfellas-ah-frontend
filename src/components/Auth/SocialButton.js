import React from 'react';

const SocialButton = ({ image, className, type }) => (
  <div className="social-link">
    <a href={`http://localhost:3000/api/auth/${type}`}>
      <div className={className}>
        <img src={image} alt="" />
      </div>
    </a>
  </div>
);

export default SocialButton;
