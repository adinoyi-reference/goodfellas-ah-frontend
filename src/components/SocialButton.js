import React from 'react';

const SocialButton = ({ image, className }) => (
  <div className="social-link">
    <div className={className}>
      <img src={image} alt="" />
    </div>
  </div>
);

export default SocialButton;
