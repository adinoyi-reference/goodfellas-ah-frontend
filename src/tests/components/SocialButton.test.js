import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SocialButton from '../../components/Auth/SocialButton';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<SocialButton />);
});

afterEach(() => wrapped.unmount());

describe('SocialButton UI', () => {
  describe('render features', () => {
    test('component should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});
