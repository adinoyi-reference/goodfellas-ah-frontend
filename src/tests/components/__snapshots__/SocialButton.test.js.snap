// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialButton UI render features component should render as expected 1`] = `
<div
  className="social-link"
>
  <a
    href="http://localhost:3000/api/auth/undefined"
  >
    <div>
      <img
        alt=""
      />
    </div>
  </a>
</div>
`;
